packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -259,15 +257,36 @@ abstract class ImplicitlyAnimatedWidget extends StatefulWidget {
 ///
 /// This is the type of one of the arguments of [TweenVisitor], the signature
 /// used by [AnimatedWidgetBaseState.forEachTween].
+///
+/// Instances of this function are expected to take a value and return a tween
+/// beginning at that value.
+/// beginning at that value.
 typedef TweenConstructor<T> = Tween<T> Function(T targetValue);
 
-/// Signature for callbacks passed to [AnimatedWidgetBaseState.forEachTween].
+/// Signature for callbacks passed to [ImplicitlyAnimatedWidgetState.forEachTween].
+///
+/// [ImplicitlyAnimatedWidgetState.forEachTween] will be passed an instance of
+/// this visitor and is expected to call the visitor with the appropriate
```

Comment:
nit: maybe split this paragraph into two sentences to make i easier to read and comprehend?